packages/reva-api/infra/graphql/candidacy/index.ts

```diff
@@ -205,6 +205,8 @@ export const resolvers = {
       const result = await submitCandidacy({
         updateCandidacyStatus: candidacyDb.updateCandidacyStatus,
         getCandidacyFromId: candidacyDb.getCandidacyFromId,
+        existsCandidacyHavingHadStatus:
+          candidacyDb.existsCandidacyHavingHadStatus,
```

Comment:
J'ai une erreur ici
Property 'existsCandidacyHavingHadStatus' does not exist on type 'typeof import("reva/packages/reva-api/infra/database/postgres/candidacies")'.
```

Reply:
update : pas de soucis, c'était dû au cherry pick de cette PR sur une branche de release